### ISSUE TITLE
ContentConsistencyVerifier bug

### ISSUE THREAD

**Issue by danielgomezrico** [mikepenz/MaterialDrawer#109]
I´m getting this every time I open the drawer, why?
Code:
```java
drawer = new Drawer().withActivity(this)
//		  .withToolbar(toolbar)
			.withOnDrawerItemClickListener(this)
			.withHeader(ResourceHelper.getImageView(this, R.mipmap.drawer_header))
			.addDrawerItems(
				new PrimaryDrawerItem().withName("My Packs"), //TODO: (Drawer) Move to Strings
				new SecondaryDrawerItem().withName("Get Sound Packs")
			).build();
```

Message:
```
entConsistencyVerifier  D  TouchEvent: Source was not SOURCE_CLASS_POINTER.
                        D    in android.widget.FrameLayout{1989474a V.E..... ........ 0,0-1440,2268 #7f0c0063 app:id/content_layout}
                        D    0: sent at 6470436000000, MotionEvent { action=ACTION_CANCEL, id[0]=0, x[0]=0.0, y[0]=0.0, toolType[0]=TOOL
                           _TYPE_UNKNOWN, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime
                           =6470436, downTime=6470436, deviceId=0, source=0x0 }
                        D    -- recent events --
                        D    1: sent at 6470260766000, (unhandled) MotionEvent { action=ACTION_DOWN, id[0]=0, x[0]=16.0, y[0]=573.0, too
                           lType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=
                           0, eventTime=6470260, downTime=6470260, deviceId=0, source=0x1002 }
                        D    2: sent at 6452910000000, (unhandled) MotionEvent { action=ACTION_CANCEL, id[0]=0, x[0]=0.0, y[0]=0.0, tool
                           Type[0]=TOOL_TYPE_UNKNOWN, buttonState=0, metaState=0, flags=0x80000000, edgeFlags=0x0, pointerCount=1, histo
                           rySize=0, eventTime=6452910, downTime=6452910, deviceId=0, source=0x0 }
                        D    3: sent at 6452741348000, (unhandled) MotionEvent { action=ACTION_DOWN, id[0]=0, x[0]=13.0, y[0]=718.0, too
                           lType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=
                           0, eventTime=6452741, downTime=6452741, deviceId=0, source=0x1002 }
                        D    4: sent at 6451292000000, (unhandled) MotionEvent { action=ACTION_CANCEL, id[0]=0, x[0]=0.0, y[0]=0.0, tool
                           Type[0]=TOOL_TYPE_UNKNOWN, buttonState=0, metaState=0, flags=0x80000000, edgeFlags=0x0, pointerCount=1, histo
                           rySize=0, eventTime=6451292, downTime=6451292, deviceId=0, source=0x0 }
                        D    5: sent at 6451123336000, (unhandled) MotionEvent { action=ACTION_DOWN, id[0]=0, x[0]=23.0, y[0]=678.0, too
                           lType[0]=TOOL_TYPE_FINGER, buttonState=0, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=
                           0, eventTime=6451123, downTime=6451123, deviceId=0, source=0x1002
```

**Comment by mikepenz** [comment]
@danielgomezrico seems to be a Android bug in the main android source code:
https://code.google.com/p/android/issues/detail?id=58183

No logs which are triggered on my side. Do you have any lag or something?

**Comment by danielgomezrico** [comment]
Sometimes the drawer don't drag, it try but stay showing like 10% of the drawer but even if I continue moving my finger it don't show it, but I don't know if it's related to this.

**Comment by Rainer-Lang** [comment]
Tried new version?

**Comment by mikepenz** [comment]
@danielgomezrico which version are you using. This was fixed in v2.5.0

**Comment by danielgomezrico** [comment]
yeah, it's fixed
